File: plugins/particle-connect/android/src/main/kotlin/network/particle/flutter/bridge/model/LoginData.java
package network.particle.flutter.bridge.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;


public class LoginData {

    @SerializedName("login_type")
    public String loginType;

    public String account;

    @SerializedName("support_auth_type_values")
    public List<String> supportAuthTypeValues;
}

File: particle-auth/android/src/main/kotlin/network/particle/auth_flutter/bridge/model/LoginData.kt
package network.particle.auth_flutter.bridge.model

import com.google.gson.annotations.SerializedName
import com.particle.network.service.model.LoginAuthorization

class LoginData {
    @SerializedName("login_type")
    var loginType: String? = null
    var account: String? = null

    @SerializedName("support_auth_type_values")
    var supportAuthTypeValues: List<String>? = null

    @SerializedName("social_login_prompt")
    var prompt: String? = null

    @SerializedName("login_form_mode")
    var loginFormMode: Boolean? = null

    @SerializedName("authorization")
    val authorization: LoginAuthorization? = null
}